packages/dashmate/configs/schema/configJsonSchema.js

```diff
@@ -368,8 +368,12 @@ module.exports = {
                               type: ['string', 'null'],
                               minLength: 32,
                             },
+                            privateKey: {
+                              type: ['string', 'null'],
+                              minLength: 32,
```

Comment:
This string in PEM format is many hundreds of characters, will we store it in the config here?

Reply:
The point is to be able to recover any dashmate state from config. Do you think we need to put config + this file separately?

Reply:
Removed it from the config